### ISSUE TITLE
Excel instead of CSV

### ISSUE THREAD

**Issue by moethu** [moethu/Tortuga#4]
read Excel instead of CSV

**Comment by mostaphaRoudsari** [comment]
Google spreadsheets?

**Comment by moethu** [comment]
Good point. Where the ... are you?! I tried to have dinner at the Iranian restaurant in your honor the other day but I didn't remember its name.

**Comment by mostaphaRoudsari** [comment]
Still waiting for my Visa. Here is the math problem. I lived and worked in the US for 5 years. Then I left the country for 5 days and applied for my new Visa to go back, and so far it took them 70 days to check if I can go back and do what I have been doing for 5 years. The best part is that during the 5 days that I left the country I was in the plane for 1 day and was teaching a workshop for the next 4 days! You do the math yourself. :)

PS: I get back to so many of my emails but still haven't had a chance to get to Tortuga, and LCA. Have been following your work though.
